### DISCUSSION TITLE
Simple voltage drop calculation

### DISCUSSION THREAD

**Discussion by KurtMarlein** [e2nIEE/pandapower#1488] [Q&A]
Hi,

I would like to test how I can use pandapower to run a voltage drop calculation in the LV distribution grid. I have a connection of an LV cable to a MV/LV transformer and 100 homes connected. I know the distance between the homes and for every home a know the peak consumption in kW. How can I model a "home" as a load in pandapower and what parameters do I need to specify such a load and be able to run a voltage drop calculation? thanks.

Kurt.

**Comment by jkisse** [comment]
Hi, please refer to the tutorials that explain how a [simple ](https://github.com/e2nIEE/pandapower/blob/develop/tutorials/create_simple.ipynb)and [advanced ](https://github.com/e2nIEE/pandapower/blob/develop/tutorials/create_advanced.ipynb)network can be created.
